e2e/utils/test-utils.ts

```diff
@@ -224,23 +224,36 @@ export async function findOrCreateWorkspace(page: Page, opts: {alwaysCreate?: bo
     }
   }
 
-  // workspace name not found, or none was specified
+  // Find all workspaces with OWNER role
+  const existingCards = await WorkspaceCard.findAllCards(page, WorkspaceAccessLevel.Owner);
 
-  const existingWorkspaces = await workspaceCard.getWorkspaceMatchAccessLevel(WorkspaceAccessLevel.Owner);
-  if (alwaysCreate || existingWorkspaces.length === 0) {
-    // Create new workspace
+  // Filter to include Workspaces older than 30 minutes
+  const halfHourMillisec = 1000 * 60 * 30;
+  const now = Date.now();
+  const filtered = [];
```

Comment:
I think you can do a similar refactoring here with `existingCards` and `filtered`. Create a variable like... `const olderWorkspaceCards = findAllCards().filter()`

Aside from cutting down on some lines of code, it reduces the number of variables declared. Also, a variable name like `olderWorkspaceCards` provides a little bit on context of how the cards were filtered.